rfq/manager.go

```diff
@@ -1104,25 +1107,39 @@ func (m *Manager) ComputeChannelAssetBalance(ctx context.Context,
 				aggrInfo.RemoteBalance += info.RemoteBalance
 			}
 
-			channels = append(channels, ChannelWithSpecifier{
+			_, ok := peerChanMap[openChan.PubKeyBytes]
+			if !ok {
+				peerChanMap[openChan.PubKeyBytes] =
```

Comment:
I think since this isn't a nested map, you can actually just ignore `ok` here and append to the return value, which will automatically allocate space as needed.